Form-Controls/index.html

```diff
@@ -1,28 +1,94 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
 
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
+    <main>
+      <section>
+        <div class="product-form">
+          <form>
+            <!-- write your html here-->
+            <!-- try writing out the requirements first-->
+            <div>
+              <label for="name">Enter Name:</label><br />
+              <input
+                type="text"
+                id="name"
+                name="name"
+                pattern="[a-zA-Z\s]+"
+                minlength="2"
+                maxlength="15" 
+                required
+              /><br />
+              <label for="email">Enter Email:</label><br />
+              <input
+                type="email"
+                id="email"
+                name="email"
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
+                required
+              /><br />
+            </div>
 
-</html>
\ No newline at end of file
+            <div>
+              <label>Choose your favourite color:</label><br />
+              <input
+                type="radio"
+                id="red"
+                name="fav_color"
+                value="red"
+                checked
+              />
+              <label for="red">Red</label><br />
+              <input type="radio" id="yellow" name="fav_color" value="yellow" />
+              <label for="yellow">Yellow</label><br />
+              <input type="radio" id="green" name="fav_color" value="green" />
+              <label for="green">Green</label><br />
+            </div>
+
+            <div>
+              <label for="size">Choose the size:</label>
+              <select id="size" name="size">
+                <option value="xs">XS</option>
+                <option value="s">S</option>
+                <option value="m">M</option>
+                <option value="l">L</option>
+                <option value="xl">XL</option>
+                <option value="xxl">XL</option></select
```

Comment:
Psst, here, the value is xxl but the option text says XL

Reply:
Thank you for your review Sally. I know it is silly mistake and I was rushing a bit :) I will take extra care in future.